server/subject_transform.go

```diff
@@ -69,7 +69,9 @@ type SubjectTransformer interface {
 	TransformTokenizedSubject(tokens []string) string
 }
 
-func NewSubjectTransform(src, dest string) (*subjectTransform, error) {
+func NewSubjectTransform(src, dest string, strict bool) (*subjectTransform, error) {
```

Comment:
I would recommend leaving original and adding in new call that the old one calls with default of false.

```go
func NewSubjectTransform(src, dest string) (*subjectTransform, error) {
  return NewSubjectTransformWithStrict(src, dest, false)
}
```